vllm/entrypoints/openai/api_server.py

```diff
@@ -24,6 +24,8 @@
 from fastapi.exceptions import RequestValidationError
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.responses import JSONResponse, Response, StreamingResponse
+from prometheus_client import make_asgi_app
+from prometheus_fastapi_instrumentator import Instrumentator
```

Comment:
remove?

Reply:
can't mount_metrics is added back here, so it's needed. That's why I wanted to keep `mount_metrics` entirely in prometheus.